browser/components/CodeEditor.js

```diff
@@ -204,6 +208,29 @@ export default class CodeEditor extends React.Component {
     textarea.value = `${textarea.value.substr(0, textarea.selectionStart)}${imageMd}${textarea.value.substr(textarea.selectionEnd)}`
   }
 
+  handlePaste (editor, e) {
+    const dataTransferItem = e.clipboardData.items[0]
+    if (!dataTransferItem.type.match('image')) return
+
+    const blob = dataTransferItem.getAsFile()
+    let reader = new FileReader()
+    let base64data
+
+    reader.readAsDataURL(blob)
+    reader.onloadend = () => {
+      base64data = reader.result.replace(/^data:image\/png;base64,/, '')
```

Comment:
Here's the jpg file I pasted from the clipboard.
![image](https://user-images.githubusercontent.com/11307908/29548902-8b722898-873f-11e7-8b77-c4d06d490db2.png)